modules/cr_cards/config/install/field.field.paragraph.cr_cards.field_cr_cards_background.yml

```diff
@@ -10,7 +10,7 @@ id: paragraph.cr_cards.field_cr_cards_background
 field_name: field_cr_cards_background
 entity_type: paragraph
 bundle: cr_cards
-label: 'Cards row Background colour'
```

Comment:
Are we renaming terminology here?

Reply:
We're not very consistent in using the term row for every paragraph. In this case it makes sense to not call it a row as it's a grid and can spill over to 2 lines if you 5 cards for example.

Reply:
Ok cool, makes sense; just need to make sure Alex et al knows this